src/plugins/intel_cpu/src/nodes/executors/kleidiai/kleidiai_mm.cpp

```diff
@@ -31,7 +31,21 @@ static std::vector<T> normalizeDimsTo2D(const std::vector<T>& dims) {
     return {std::accumulate(dims.begin(), dims.end() - 1, (T)1, std::multiplies<T>()), dims[dims.size() - 1]};
 }
 
+static bool useDynamicQuantizationImpl(const FCAttrs& attrs, const MemoryDescPtr& weightDesc) {
+    if (attrs.dynamicQuantizationGroupSize != UINT64_MAX)
+        return false;
```

Comment:
To indicate that the group quantization is disabled, group size "0" is used.
Please refer to: https://github.com/openvinotoolkit/openvino/blob/master/src/inference/include/openvino/runtime/properties.hpp#L585-L597
Please align the check with the common behavior. Here and in all the relevant places.

Reply:
Agreed to keep it as it is.